### PR TITLE
Kb for auxiliary/scanner/http/open_proxy

### PR DESCRIPTION
**Pull request by itsmeroy2012**
### Part of issue #8296

### CODE REVIEW

File: documentation/modules/auxiliary/scanner/http/open_proxy.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+## Description
+
+Checks if an HTTP proxy is open. False positive are avoided verifying the HTTP return code and matching a pattern. The CONNECT method is verified only the return code. HTTP headers are shown regarding the use of proxy or load balancer.
```
Comment (wwebb-r7) [inline documentation/modules/auxiliary/scanner/http/open_proxy.md]: > False positive are avoided verifying the HTTP return code and matching a pattern

Perhaps you meant

> False positives are avoided by verifying the HTTP return code and matching a pattern

also
>The CONNECT method is verified only the return code

maybe

> The CONNECT method is verified only by the return code

**Review by wwebb-r7 — CHANGES_REQUESTED** [review]

File: documentation/modules/auxiliary/scanner/http/open_proxy.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Description
 
-Checks if an HTTP proxy is open. False positive are avoided verifying the HTTP return code and matching a pattern. The CONNECT method is verified only the return code. HTTP headers are shown regarding the use of proxy or load balancer.
+Checks if an HTTP proxy is open. False positive are avoided by verifying the HTTP return code and matching a pattern. The CONNECT method is verified only by the return code. HTTP headers are shown regarding the use of proxy or load balancer.
```
Comment (wwebb-r7) [inline documentation/modules/auxiliary/scanner/http/open_proxy.md]: Don't forget to make "false positive" plural

**Review by wwebb-r7 — APPROVED** [review]

**Comment by wwebb-r7** [comment]
### Release Notes

This update adds documentation for scanner/http/open_proxy

**Comment by wwebb-r7** [comment]
Looks good to go.  Thanks for the quick updates.

**Comment by vipzen** [comment]
I was waiting for this. Thanks.

**Pull request merged**